emily/handler/src/database/accessors.rs

```diff
@@ -559,8 +557,11 @@ pub async fn add_chainstate_entry(
             .await
             .and_then(|existing_entry: ChainstateEntry| {
                 if existing_entry.key != entry.key {
-                    debug!("Inconsistent state because of a conflict with the current interpretation of a height.");
-                    debug!("Existing entry: {existing_entry:?} | New entry: {entry:?}");
+                    debug!(
```

Comment:
Should this be `warn!` as well?

Reply:
https://github.com/stacks-sbtc/sbtc/pull/1627/commits/4e2bc6fe97e71dbb518da77cd03dae00ee821e43